docs/register/prop-method-injection.rst

```diff
@@ -21,7 +21,7 @@ To support :doc:`circular dependencies <../advanced/circular-dependencies>`, use
 
     builder.Register(c => new A()).OnActivated(e => e.Instance.B = e.Context.Resolve<B>());
 
-If the component is a :ref:`reflection component <register-registration-reflection-components>`, use the ``PropertiesAutowired()`` modifier to inject properties:
+If the component is a :ref:`reflection component <register-registration-reflection-components>`, use the ``PropertiesAutowired()`` modifier to inject properties. Using ``PropertiesAutoWired()`` will inject properties that are *writable* and *public*:
```

Comment:
Is it worth mentioning the `IPropertySelector` overload, or saying "By default, using `PropertiesAutoWired()` will inject properties that are writeable and public"? Or something?

Reply:
I added more to the example to show default, delegate, and custom selector uses.